CONTRIBUTING.rst

```diff
@@ -99,7 +98,7 @@ Before you submit a pull request, check that it meets these guidelines:
 2. If the pull request adds functionality, the docs should be updated. Put
    your new functionality into a function with a docstring, and add the
    feature to the list in README.rst.
-3. The pull request should work for Python 3.7 to 3.11, and PyPy. Check
+3. The pull request should work for Python 3.8 to 3.11, and PyPy. Check
```

Comment:
This type of thing will probably fall between the cracks and get outdated, what if we change it to something like currently supported pyhton versions , and PyPy?

Reply:
I agree. This should be more future-proof.